modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/40_token_filters.yml

```
@@ -1198,6 +1198,43 @@
     - match:  { tokens.0.token: foo }
 
 ---
+"delimited_termfreq":
```

We very likely need version check here:

```suggestion
"delimited_termfreq":
  - skip:
      version: " - 2.9.99"
      reason:  "elimited_termfreq token filter was added in v2.10.0"

```

---

done